vitest.config.js

```diff
@@ -2,18 +2,21 @@ import { defineConfig, mergeConfig } from 'vitest/config'
 
 import viteConfig from './vite.config'
 
-export default mergeConfig(viteConfig, defineConfig({
-  test: {
-    globals: true,
-    environment: 'jsdom',
-    setupFiles: './test/unit/setup.js',
-    include: [
-      'src\/**\/*.{test,spec}.{js,mjs,cjs,ts,mts,cts,jsx,tsx}'
-    ],
-    deps: {
-      inline: [
-        'ipld-explorer-components'
-      ]
+export default defineConfig((configEnv) => mergeConfig(
+  viteConfig(configEnv),
+  defineConfig({
+    test: {
+      globals: true,
+      environment: 'jsdom',
+      setupFiles: './test/unit/setup.js',
+      include: [
+        'src/**/*.{test,spec}.{js,mjs,cjs,ts,mts,cts,jsx,tsx}'
+      ],
+      deps: {
+        inline: [
+          'ipld-explorer-components'
+        ]
+      }
     }
-  }
-}))
+  })
+))
```

Comment:
clearing out package-lock.json and refreshing it caused an error here.. vitest must've updated their api (they're still pre v1)